raster/rgb-renderer/README.md

```diff
@@ -1,31 +1,37 @@
-# Rgb Renderer
+# RGB renderer
 
-Apply an RGB renderer to a raster.
+Apply an RGB renderer to a raster layer to enhance feature visibility.
 
-An RGB renderer is used to adjust the color bands of a multispectral image.
+![Image of RGB renderer](RgbRenderer.png)
 
-![](RgbRenderer.png)
+## Use case
+
+An RGB renderer is used to adjust the color bands of a multispectral image. Remote sensing images acquired from satellites often contain values representing the reflection of multiple spectrums of light. Changing the RGB renderer of such rasters can be used to differentiate and highlight particular features that reflect light differently, such as different vegetation types, or turbidity in water.
 
 ## How to use the sample
 
-Choose one of the stretch parameter types. The other options will adjust based on the chosen type. Add your inputs and press the Update button to update the renderer.
+Choose one of the stretch parameter types. The other options will adjust based on the chosen type. Add your inputs and select the 'Render' button to update the renderer.
 
 ## How it works
 
-To apply a `RgbRenderer` to a `RasterLayer`:
-
-1. Create a `Raster` from a multispectral raster file
-2. Create a `RasterLayer` from the raster
-3. Create a `Basemap` from the raster layer and set it to the map
-4. Create a `RgbRenderer`, specifying the stretch parameters and other properties
-5. Set the renderer on the raster layer with `rasterLayer.setRenderer(renderer)`
+1. Create a `Raster` from a multispectral raster file.
+2. Create a `RasterLayer` from the raster.
+3. Create a `Basemap` from the raster layer and instantiate an ArcGISMap with it.
+4. Create a `RGBRenderer`, specifying the `StretchParameters` and other properties.
+5. Set the `Renderer` on the raster layer with `rasterLayer.setRasterRenderer(renderer)`.
```

Comment:
We've put `RGBRenderer` in backticks in step 4, so it's probably fine to just write renderer regularly in this step. We do the same in Stretch Renderer

```suggestion
5. Set the renderer on the raster layer with `rasterLayer.setRasterRenderer(renderer)`.
```